assets/js/components/BatchEdit/About/CoreMetadata.jsx

```diff
@@ -43,59 +42,57 @@ const BatchEditAboutCoreMetadata = ({ errors, register, ...restProps }) => {
           />
         </UIFormField>
       </div>
-      <div className="column is-half">
-        {/* Description */}
-        <UIFormField label="Description" required>
-          <UIFormTextarea
-            register={register}
-            required
-            name="description"
-            label="Description"
-            data-testid="description"
-            errors={errors}
-          />
-        </UIFormField>
-      </div>
 
-      <div className="column is-half">
-        <UIFormField label="Rights Statement">
-          <UIFormSelect
-            register={register}
-            name="rightsStatement"
-            label="Rights Statement"
-            options={rightsStatementsData ? rightsStatementsData.codeList : []}
-            errors={errors}
-            data-testid="rights-statement"
-            showHelper
-          />
-        </UIFormField>
+      <div className="column is-full">
+        {/* Alternate Title */}
+        <UIFormFieldArray
+          register={register}
+          control={control}
+          name="alternateTitle"
+          data-testid="alternate-title"
+          label="Alternate Title"
+          errors={errors}
+        />
       </div>
       <div className="column is-half">
         {/* Date Created */}
-        <UIFormField label="Date Created" notLive>
+        <UIFormField label="Date Created" required notLive>
```

Comment:
Does this need to be required?

Reply:
I think so from the docs..
<img width="395" alt="Screen Shot 2020-08-03 at 12 54 27 PM" src="https://user-images.githubusercontent.com/14085957/89212096-81e8af00-d588-11ea-9bf8-c03f47bf2c93.png">

Reply:
Nothing is required until the point when the work is published. 

Date created is not yet implemented on the backend. (it's an EDTF date)